packages/eds-core-react/src/components/Textarea/Textarea.docs.mdx

```diff
@@ -0,0 +1,28 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Textarea } from '.'
+
+# Textarea
+
+Helper component - is used for collecting large amounts of textual data.
```

Comment:
Should maybe make it more clear this is mostly internal and most user should just use TextField

"Helper component developed mainly for internal use. Can be used as a building block for users requiring more flexibility than TextField provides" 
Something like this?

Reply:
Updated